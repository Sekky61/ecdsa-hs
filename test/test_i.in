Curve {
  p: 23
  a: 0
  b: 1
  g: Point {
    x: 5
    y: 1
  }
  n: 7
  h: 1
}
