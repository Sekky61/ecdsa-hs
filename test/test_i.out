Curve {
p: 0x17
a: 0
b: 1
g: Point {
   x: 0x5
   y: 0x1
}
n: 0x7
h: 1
}
